code/modules/mob/living/carbon/alien/alien_attacks.dm

```diff
@@ -28,7 +28,7 @@
 			playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
 			for(var/mob/O in viewers(src, null))
 				if ((O.client && !( O.blinded )))
-					O.show_message(text("<span class='warning'>[] has grabbed [] passively!</span>", M, src), 1)
+					O.show_message("<span class='warning'>[M] has grabbed [src] passively!</span>", 1)
```

Comment:
Maybe `is_invisible_to` should be checking for blindness, so that you can fix this `for` too? And others in this file.